pdns/recursordist/docs/lua-config/rpz.rst

```diff
@@ -112,6 +112,14 @@ tags
 
 List of tags as string, that will be added to the policy tags exported over protobuf when a policy of this zone matches.
 
+overridesGettag
+^^^^^^^^^^^^^^^
+.. versionadded:: 4.4.0
+
+`gettag_ffi` can set an answer to a query.
+By default an RPZ hit overrides this answer, unless the policy is `rpz-passthru` or this option is set to `false`.
```

Comment:
I wonder if we really want to ignore `rpz-passthru` matches, which are a "whitelisting" mechanism? Would it make sense to only apply the answer set from `gettag_ffi` if there was no match or if this option is set to `false`?

Reply:
Yes, this is open for discussion. I'll sleep a night on it :-)